frontend/javascripts/oxalis/view/right-border-tabs/meshes_view.js

```diff
@@ -458,14 +473,17 @@ class MeshesView extends React.Component<Props, State> {
       );
     };
 
-    const getHeaderDropdownMenu = () => (
-      <Menu>
-        <Menu.Item onClick={startComputingMeshfile} disabled={!features().jobsEnabled}>
-          {getComputeMeshfileTooltip("Compute Mesh File")}
-        </Menu.Item>
-        <Menu.Item>{getStlImportItem()}</Menu.Item>
-      </Menu>
-    );
+    const getHeaderDropdownMenu = () => {
+      const { disabled, title } = getComputeMeshFileTooltipInfo();
+      return (
+        <Menu>
+          <Menu.Item onClick={startComputingMeshfile} disabled={disabled}>
+            <Tooltip title={title}>Compute Mesh File</Tooltip>
```

Comment:
I am also not a big fan of the button title. What is a `mesh file`? This is very technical. How about something more descriptive. "Computes Mesh for (whole) dataset" or "Pre-compute meshes for all segments" or "Compute all meshes". Or any such combination.

 I find it hard to find the difference between the button "Compute Mesh" and "Compute Mesh File". Likely the other button should be called something like "Compute for Mesh for (active) Segment" or "Compute single mesh" or "compute temporay mesh" ...

Reply:
@normanrz Any comments?

Reply:
I don't have a strong opinion re `mesh file`.
I agree that meshes vs isosurfaces is quite confusing. They are essentially the same but have different names.

Reply:
I changed the button labels to:
- Precompute Meshes (tooltip: Precompute meshes for all segments of this dataset so that meshes for segments can be loaded quickly afterwards from a mesh file.)
- Load Precomputed Mesh
- Compute Mesh (ad hoc)

I hope this makes it clearer. I also hide the "Compute Mesh (ad hoc)" button automatically, if a mesh file is already available (since there is no benefit in using the ad-hoc part).

> I agree that meshes vs isosurfaces is quite confusing. They are essentially the same but have different names.

I went through the code and changed the last user-facing "isosurface" bits to "mesh", but there were quite rare (mainly, in the download/reload buttons of the mesh list). I hope I caught them all.